addons/loyalty/views/loyalty_program_views.xml

```diff
@@ -105,7 +105,7 @@
                             <label for="limit_usage" invisible="program_type in ('gift_card', 'ewallet')"/>
                             <span invisible="program_type in ('gift_card', 'ewallet')">
                                 <field name="limit_usage" class="oe_inline"/>
-                                <span invisible="not limit_usage"> to <field name="max_usage" class="oe_inline"/> usages</span>
+                                <span invisible="not limit_usage"> <field name="total_order_count" class="oe_inline" readonly="1"/> / <field name="max_usage" class="oe_inline"/> usages</span>
```

Comment:
This is a bit strange and misleading, so I asked loyalty's PO, and he thinks this would be better: 
![image](https://github.com/user-attachments/assets/520228a7-c693-43ab-9b04-7c55d8f127b8)